pyflagser/tests/test_flagio.py

```diff
@@ -1,23 +1,43 @@
 """Testing for the python bindings of the C++ flagser library."""
 
+import os
+import shutil
+from tempfile import mkdtemp
+from urllib.request import urlopen
+
 import numpy as np
-import scipy.sparse as sp
 import pytest
-import os
+import scipy.sparse as sp
 from numpy.testing import assert_almost_equal
 
 from pyflagser import loadflag, saveflag
 
-flag_files = []
-
-dirname = os.path.join(os.path.dirname(__file__), "../../flagser/test")
-for file in os.listdir(dirname):
-    if file.endswith(".flag"):
-        flag_files.append(os.path.join(dirname, file))
+try:
+    dirname = os.path.join(os.path.dirname(__file__), "../../flagser/test")
+    list_dir = os.listdir(dirname)
+    flag_files = [os.path.join(dirname, fname)
+                  for fname in os.listdir(dirname)
+                  if fname.endswith(".flag")]
+    download_files = False
+except FileNotFoundError:
+    # Download from remote bucket
+    temp_dir = mkdtemp()
+    bucket_url = 'https://storage.googleapis.com/l2f-open-models/giotto-tda' \
```

Comment:
Those files are all in the git repos of luetge/flagser. I don't think we need to host them on a company's server.

Reply:
In practice, it is easier this way for now. It can be in any case changed later. Furthermore, the files are tiny and available to everyone from those URLs.